src/transfer/publish/config.vsh.yaml

```diff
@@ -1,33 +0,0 @@
-name: publish
-namespace: "transfer"
-description: Publish an artifact and optionally rename with parameters
-authors:
-  - __merge__: /src/authors/toni_verbeiren.yaml
-    roles: [ maintainer ]
-arguments:
-  - name: "--input"
-    alternatives: ["-i"]
-    type: file
```

Comment:
where does it say this was deprecated? :raised_eyebrow:

Reply:
https://github.com/openpipelines-bio/openpipeline/pull/941